h/static/styles/help-page.scss

```diff
@@ -1,10 +1,10 @@
 $base-font-size: 16px;
 $base-line-height: 22px;
 
-@import './reset';
-@import './common';
-@import './elements';
-@import './mixins/icons';
+@import 'reset';
```

Comment:
Agreed, but relative imports are horrible in CSS anyway. In #3610 and soon another PR all imports will be absolute and as a result hopefully much clearer.